src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -43,17 +46,17 @@
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_GIT_USERNAME + "GITHUB_USERNAME]...\n"
-            + "[" + PREFIX_TAG + "TAG] "
+            + "[" + PREFIX_GIT_USERNAME + "GITHUB_USERNAME] "
+            + "[" + PREFIX_TAG + "TAG]...\n "
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com "
             + PREFIX_GIT_USERNAME + "john123";
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person with these exact same parameters already "
-            + "exists in the address book.";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.\n"
+            + MESSAGE_USAGE;
+    public static final String MESSAGE_DUPLICATE_PERSON = "At least one field of this contact must be edited! \n%1$s";
```

Comment:
The message does not seems to match with the variable name🤔 (ps. can ignore if is intended)